lib/resolvers/project.js

```diff
@@ -103,5 +103,19 @@ module.exports = ({ models }) => ({ action, data, id, meta }, transaction) => {
       .then(() => Project.queryWithDeleted(transaction).findById(id));
   }
 
+  if (action === 'delete-amendments') {
+    return ProjectVersion.query(transaction).where({ projectId: id })
+      .then(versions => {
+        const granted = versions.find(v => v.status === 'granted');
+        return versions.filter(v => v.status === 'draft' && v.createdAt > granted.createdAt);
```

Comment:
What about submitted drafts?

Reply:
Do submitted drafts get changed to "withdrawn" when you withdraw a task? I think we can safely remove those, but you shouldn't be able to delete "submitted" (and the button doesn't show in the UI if there's an open task anyway).

Reply:
no, they stay as submitted _forever..._